### ISSUE TITLE
Make numpad work better with numerical habits

### ISSUE THREAD

**Issue by hiqua** [iSoron/uhabits#900]
## Pre-submission checklist
- [X] I am submitting a bug report, not a feature request.
- [X] I am running the latest version of Loop Habit Tracker.
- [X] I have have searched for similar issues, but did not find any matches.

## Description

The user can enter a new entry using the numpad. But using the `.` doesn't do anything, and it seems that the fractional part cannot be entered with the numpad.

Ideally, typing `3.15` on the numpad should enter exactly this value.

## Steps to reproduce
1. add new entry for a numerical habit
2. enter one digit
3. enter `.`
4. you're still at the integer part instead of the fractional part

## System information
 - v2.0.1

**Comment by iSoron** [comment]
Thanks for bringing this up. This is indeed an issue. PRs are welcome.

**Comment by sumanabhi** [comment]
Hi @iSoron ,
I would love to work on this issue.

**Comment by hiqua** [comment]
@sumanabhi feel free!

**Comment by sumanabhi** [comment]
hey @hiqua 
One thing, I just want to confirm that the User can either enter `.95` or we have to restrict the user programmatically to enter the decimal only after entering one number e.g. `3.15`.

if our code can understand in other places of showing this data to the user automatically or we have to programmatically manage this as well.

**Comment by iSoron** [comment]
Closing this in favor of #922, which explains the problem more clearly.
